stdlib/public/Concurrency/Clock.cpp

```diff
@@ -87,18 +89,33 @@ void swift_get_time(
       *seconds = suspending.tv_sec;
       *nanoseconds = suspending.tv_nsec;
 #elif defined(_WIN32)
-      LARGE_INTEGER freq;
-      QueryPerformanceFrequency(&freq);
-      LARGE_INTEGER count;
-      QueryPerformanceCounter(&count);
-      *seconds = count.QuadPart / freq.QuadPart;
-      if (freq.QuadPart < 1000000000) {
-        *nanoseconds = 
-            ((count.QuadPart % freq.QuadPart) * 1000000000) / freq.QuadPart;
+      // QueryUnbiasedInterruptTimePrecise() was added in Windows 10 and is, as
+      // the name suggests, more precise than QueryUnbiasedInterruptTime().
+      // However, despite being declared in Windows' headers, we must look it up
+      // dynamically at runtime.
```

Comment:
I assume this is because we don't have Windows 10 as a minimum target? (That is, `_WIN32_WINNT` would need to be defined to something else?)

Perhaps the comment should say _why_ it needs to be dynamically looked-up.

Reply:
Exactly. I'll edit the comment to make the connection to Windows 10 clearer.

Reply:
I tried at desk, and from what I can see, even on Windows 10 this symbol is not exported normally. I don't need to change `_WIN32_WINNT`, the symbol is _declared_, but it's not available at link time. It is available when dynamically looking it up using `GetProcAddress()`, and it's fully documented on MSDN. I'll note the issue in the comment.

Reply:
If it's not available at link time, it might just be that we don't link against the relevant `.lib` file?

Reply:
Ah, I'm not crazy! swift-corelibs-dispatch has the same problem. See [this](https://github.com/apple/swift-corelibs-libdispatch/pull/498) PR from that repo.